localstack/utils/persistence.py

```diff
@@ -7,13 +7,13 @@
 import logging
 from six import add_metaclass
 from abc import ABCMeta, abstractmethod
-from localstack.config import DATA_DIR
+from localstack.config import DATA_DIR, is_env_true
 from localstack.utils.aws import aws_stack
 from localstack.utils.common import to_bytes, to_str
 from localstack.utils.bootstrap import is_api_enabled
 from localstack.services.generic_proxy import ProxyListener
 
-USE_SINGLE_DUMP_FILE = True
+USE_SINGLE_DUMP_FILE = is_env_true('USE_SINGLE_DUMP_FILE')
```

Comment:
Thanks for this PR @collinsauve 👍 . If we want to make this `True` by default, then I think this should be changed to:
```
USE_SINGLE_DUMP_FILE = is_env_not_false('USE_SINGLE_DUMP_FILE')
```

Reply:
oof.  now I have to rethink how I tested this :suspect: 

Definitely will fix that.  Do you want to keep the env variable name?  If so I will rewrite the commit.

Reply:
Thanks @collinsauve I'd suggest renaming this to `PERSISTENCE_SINGLE_FILE`, to be a bit more in line with the naming scheme of the other config variables.. Thanks

Reply:
Btw - let's also add this to `CONFIG_ENV_VARS` in config.py: https://github.com/localstack/localstack/blob/master/localstack/config.py#L227

Reply:
Sry for the delay @whummer.   Rewrote it with that variable, should be good for another review.